src/main/java/com/laytonsmith/tools/Interpreter.java

```diff
@@ -850,7 +850,8 @@ public static void install() {
 		if(null == OSUtils.GetOS()) {
 			StreamUtils.GetSystemErr().println("Cmdline MethodScript is only supported on Unix and Windows");
 			return;
-		} else switch(OSUtils.GetOS()) {
+		}
+		switch(OSUtils.GetOS()) {
```

Comment:
This happens to be ok in this case, but one cannot do this in general, as this is slightly different behavior than how it was.

Reply:
@LadyCailin Can you explain the different behavior? I would think it's equivalent because of the return.